.github/workflows/build.yaml

```diff
@@ -103,12 +103,21 @@ jobs:
           zstdcat build.tar.zst | tar x
           cd build
           cpack
+      - name: Install dev package
+        run: |
+          apt update
+          export DEBIAN_FRONTEND=noninteractive
+          export TZ="America/New_York"
+          apt install -y ./build/leap_*.deb
+          apt install -y ./build/leap-dev*.deb
```

Comment:
I want to keep the workflow as concise and clutter free as possible, so two changes I'd like here:

1. move the `DEBIAN_FRONTEND` and `TZ` to the Dockerfile
2. combine the two installs, either as `./build/leap_*.deb ./build/leap-dev*.deb` or `./build/leap*.deb` or whatever